packages/ng/core/src/lib/type/style.ts

```diff
@@ -0,0 +1,9 @@
+/**
+ * Available CSS palettes
+ */
+export type Palette = 'success' | 'warning' | 'error' | 'primary' | 'grey' | 'none';
+
+/**
+ * Size that accepts only medium or small
+ */
+export type SizeMS = 'm' | 's';
```

Comment:
Reading the PR that moves all files, I saw this type that is not used anymore

Reply:
Indeed, I'll make the change in #2210 though, to make sure it doesn't create new conflicts 😄